core-plugins/docs/FTP-batchsource.md

```diff
@@ -0,0 +1,40 @@
+# FTP/SFTP Batch Source
```

Comment:
I would also prefer calling it SFTP Batch Source. I think the use case for SFTP might be greater than FTP source. @sreevatsanraman Any thoughts?